mkdocs.yml

```diff
@@ -37,7 +37,7 @@ markdown_extensions:
   - md_in_html # Used for grids
 
 extra_javascript:
-  - javascripts/katex.js
+  - js/katex.js
```

Comment:
This `javascripts`subdirectory was requested by Material for Mkdocs.

Reply:
I think it's convention to use `js`, I renamed/formatted the file `katex.js` as well.